Core/DefaultAbstractController.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+use Twig\Environment;
+use Twig\Loader\FilesystemLoader;
+
+require_once PROJECT_CORE . 'DefaultControllerInterface.php';
+
+abstract class DefaultAbstractController implements DefaultControllerInterface
+{
+    private $request;
+    private static $twig = null;
+
+    public function getRequest()
```

Comment:
il y a un soucis de conception, tu peux pas dans un getter faire un $this->request = ;
c'est un setter qui affecte une valeur a une propriete de classe, ton getter devrai juste te retourner getInstance